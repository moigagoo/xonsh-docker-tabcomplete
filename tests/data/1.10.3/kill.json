{
        "arguments": [
                "--help",
                "-s", "--signal=KILL"
        ],
        "commands": []
}
