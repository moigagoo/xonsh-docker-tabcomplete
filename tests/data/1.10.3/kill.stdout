
Usage:	docker kill [OPTIONS] CONTAINER [CONTAINER...]

Kill a running container

  --help               Print usage
  -s, --signal=KILL    Signal to send to the container
